### DISCUSSION TITLE
Support for JWKS?

### DISCUSSION THREAD

**Discussion by jordaniversen** [golang-jwt/jwt#179] [Q&A] [answered]
Is there native support for handling JWKS?

Thank you

**Comment by oxisto** [comment, marked as answer]
Hi @jordaniversen. No, not at the moment.

I would recommend the excellent library https://github.com/MicahParks/keyfunc from @MicahParks. We should probably add this to the list of extensions in the README. There was a short discussion whether we should implement support for JWKS, but for now we decided not to include it in the main repository. Maybe we can think about adding a separate project under the `golang-jwt` umbrella.

**Reply by MicahParks** [reply]
There now exists https://github.com/MicahParks/jwkset, which is primarily for server-side JWK Set assets at the moment. (`keyfunc` is a JWK Set client.)
